apps/main/src/dex/hooks/usePoolAlert.tsx

```diff
@@ -11,6 +11,28 @@ import { ExternalLink, InternalLink } from '@ui/Link'
 import { breakpoints } from '@ui/utils'
 import { shortenAddress } from '@ui-kit/utils'
 
+// Builder for CrossCurve labels
+const CrossCurveLabel = {
+  mint(pool: string) {
+    return {
+      label: 'Mint s-tokens',
+      url: `https://app.crosscurve.fi/liquidity?pool=${pool}&action=deposit&type=curve`,
+    }
+  },
+  redeem(pool: string) {
+    return {
+      label: 'Redeem s-tokens',
+      url: `https://app.crosscurve.fi/liquidity?pool=${pool}&action=withdraw&type=curve`,
+    }
+  },
+  swap(inputChainId: number, inputToken: string, outputChainId: number, outputToken: string) {
```

Comment:
It would be nice to use `Chain` enum or an object for readability
```suggestion
  swap(inputChainId: Chain, inputToken: string, outputChainId: Chain, outputToken: string) {
```
or
```suggestion
  swap(params: { inputChainId: Chain, inputToken: string, outputChainId: Chain, outputToken: string }) {
```